etl/nifi/conf/bootstrap-aws.conf

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
Why do we need an AWS config file?

Reply:
These are some config file which I copied directly from the nifi container, so that we can persist the changes. Not all config file will be required, but while copying just copied everything to the repo. Please let me know if you want me to remove it.